src/libpcp_web/src/query_parser.y

```diff
@@ -524,6 +526,50 @@ func_sid
 		  lp->yy_np->left = $3;
 		  $$ = lp->yy_series.expr = lp->yy_np;
 		}
+	| L_NTH_PERCENTILE_INST L_LPAREN sid_vec L_COMMA L_INTEGER L_RPAREN
```

Comment:
Should the L_INTEGER in this line be "integer" like in your topk branch?  (similar case for lines 540, 551, 562 below)